ogambakerubo/DeliverIT2
# Ft get user parcels #162683861

#### What does this PR do?
Initializes the DeliverIT2 web app with a retrieve user parcels api endpoint

#### Description of Task to be completed?
Have the `run.py` working

#### How should this be manually tested?
For the API endpoint to be tested, ensure the local development server is running then use Postman or curl to test your endpoints.

Clone the repository into your local environment

```
$ https://github.com/ogambakerubo/DeliverIT2.git
```

Switch to the branch ft-get-user-parcels-#162683861
```
$ git checkout ft-get-user-parcels-#162683861
```

Change directory into DeliverIT2
```
cd DeliverIT2
```

Run `python run.py` file in your terminal

#### Any background context you want to provide?


#### What are the relevant pivotal tracker stories?
#162683861

#### Screenshots (if appropriate)
![screenshot from 2019-01-18 13-13-26](https://user-images.githubusercontent.com/43958047/51384301-fb4ab900-1b2c-11e9-91c9-b6003812d6a7.png)
